### PR TITLE
Fixes and formatting

### PR DESCRIPTION
**Pull request by TheQuantumPhysicist**

### CODE REVIEW

**Comment by TheQuantumPhysicist** [comment]
FYI: The formatting issues @iljakuklic mentioned in the last PR are due to nightly.

File: .github/workflows/code_checks.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    name: Clippy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: rustup component add clippy
+
+      # Checks enabled everywhere, including tests, benchmarks
+      - uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (muursh) [inline .github/workflows/code_checks.yml:18]: Does this exist? Looking at the repo I can't see it

Comment (TheQuantumPhysicist) [reply .github/workflows/code_checks.yml:18]: Nope. That's the wonder of copy/paste :D

Comment (TheQuantumPhysicist) [reply .github/workflows/code_checks.yml:18]: For some reason it doesn't work without it. I have no idea how CI works fully.

File: lmdb-sys/src/bindings.rs [hunk 2]
```diff
@@ -1,57 +1,57 @@
 /* automatically generated by rust-bindgen */
```
Comment (iljakuklic) [inline lmdb-sys/src/bindings.rs:1]: Should this file be formatted/modified despite its claim to be auto-generated here?

**Review by iljakuklic — APPROVED** [review]

File: lmdb-sys/src/bindings.rs (hunk 2 above)
Comment (TheQuantumPhysicist) [reply lmdb-sys/src/bindings.rs:1]: Well, they actively avoided formatting it, and it looks fine. That's why I added an exception for it.

**Review by muursh — APPROVED** [review]

**Pull request merged**